evap/evaluation/models.py

```diff
@@ -810,10 +810,16 @@ class Meta:
 
 class UserProfileManager(BaseUserManager):
     def get_queryset(self):
-        return super().get_queryset().exclude(username=UserProfile.CRONJOB_USER_USERNAME)
+        return (super().get_queryset()
+            .exclude(username=UserProfile.CRONJOB_USER_USERNAME)
+            .exclude(is_active=False)
+        )
 
     def with_cronjob_user(self):
-        return super().get_queryset()
+        return super().get_queryset().exclude(is_active=False)
+
+    def with_inactive_users(self):
+        return super().get_queryset().exclude(username=UserProfile.CRONJOB_USER_USERNAME)
```

Comment:
following proposal:

this manager gets two methods, `cronjob_user`, returning the one user (remove the method in UserProfile then), and `exclude_inactive_users`, doing the filtering.